### PR TITLE
feat(telemetry): add telemetry events to the settings page

### PR DESCRIPTION
**Pull request by aditya-radhakrishnan**
## Checklist
- [x] The PR conforms to DataHub's [Contributing Guideline](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [x] Links to related issues (if applicable)
- [x] Tests for the changes have been added/updated (if applicable)
- [x] Docs related to the changes have been added/updated (if applicable). If a new feature has been added a Usage Guide has been added for the same.
- [x] For any breaking change/potential downtime/deprecation/big changes an entry has been made in [Updating DataHub](https://github.com/datahub-project/datahub/blob/master/docs/how/updating-datahub.md)

### CODE REVIEW

File: datahub-web-react/src/app/identity/user/ViewResetTokenModal.tsx [hunk 1]
```diff
@@ -88,14 +116,7 @@ export default function ViewResetTokenModal({ visible, userUrn, username, onClos
...
                     onClick={() => {
-                        createNativeUserResetToken({
-                            variables: {
-                                input: {
-                                    userUrn,
-                                },
-                            },
-                        });
-                        setHasGeneratedResetToken(true);
+                        createNativeUserResetToken();
```
Comment (chriscollins3456) [inline datahub-web-react/src/app/identity/user/ViewResetTokenModal.tsx:119]: nit: can just be `onClick={createNativeUserResetToken}`

File: datahub-web-react/src/app/settings/Preferences.tsx [hunk 2]
```diff
@@ -69,7 +70,7 @@ export const Preferences = () => {
                 <Card>
                     <UserSettingRow>
                         <span>
-                            <SettingText>Show simplified hompeage </SettingText>
+                            <SettingText>Show simplified homepage </SettingText>
```
Comment (chriscollins3456) [inline datahub-web-react/src/app/settings/Preferences.tsx:73]: nice

**Review by chriscollins3456 — APPROVED** [review]
Looks good!

File: datahub-web-react/src/app/identity/user/ViewResetTokenModal.tsx (hunk 1 above)
Comment (aditya-radhakrishnan) [reply datahub-web-react/src/app/identity/user/ViewResetTokenModal.tsx:119]: Good call! Fixed

**Pull request merged**